### PR TITLE
Show all currently-implemented charsets with default collations

### PR DESCRIPTION
**Pull request by Hydrocharged**

### CODE REVIEW

File: sql/encodings/utf16_bin.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+package encodings
+
+// Utf16_bin_RuneWeight returns the weight of a given rune based on its relational sort order from
+// the `utf16_bin` collation.
+func Utf16_bin_RuneWeight(r rune) int32 {
+	weight, ok := utf16_bin_Weights[r]
+	if ok {
+		return weight
+	} else if r >= 0 && r <= 55295 {
```
Comment (zachmu) [inline sql/encodings/utf16_bin.go:23]: What are all these magic numbers?

Comment (zachmu) [reply sql/encodings/utf16_bin.go:23]: Could use constants or comments

File: sql/encodings/utf32_bin.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+package encodings
+
+// Utf32_bin_RuneWeight returns the weight of a given rune based on its relational sort order from
+// the `utf32_bin` collation.
+func Utf32_bin_RuneWeight(r rune) int32 {
+	weight, ok := utf32_bin_Weights[r]
+	if ok {
+		return weight
+	} else if r >= 0 && r <= 55295 {
```
Comment (zachmu) [inline sql/encodings/utf32_bin.go:23]: Same comment about magic numbers

**Review by zachmu — APPROVED** [review]

File: sql/encodings/utf16_bin.go (hunk 1 above)
Comment (Hydrocharged) [reply sql/encodings/utf16_bin.go:23]: For reference for anyone that checks out this PR later: these files are all generated from https://github.com/dolthub/collation-extractor. To understand them, it's easier to look at the related structures under the `rune_comparator.go` file than to look at a generated file.

**Pull request merged**